### DISCUSSION TITLE
What to put in the destination field?

### DISCUSSION THREAD

**Discussion by Alex-joomla** [jcwillox/hass-auto-backup#136] [Q&A]
Hey Folks,
it took me hours to find out, where the mountpoint of the Network Share, added in the frontend (Network Storage), is. It’s only visible in the Filesystem of the supervisor (I’m running on HAOS) and is there in

<img width="700" alt="Bildschirmfoto 2024-05-25 um 13 29 46" src="https://github.com/jcwillox/hass-auto-backup/assets/32268233/e9098ee0-b1dd-4fae-9f45-0fc738ea7cc2">

Question is now, what is the correct path, if i try to use a backup automation.

Config looks like this:

<img width="587" alt="Bildschirmfoto 2024-05-25 um 13 26 14" src="https://github.com/jcwillox/hass-auto-backup/assets/32268233/ad99fc2b-2ef0-446c-b054-faf1ebbc1c0a">

and it works if I do a manual Backup, but I’m using auto-backup and I don’t know what I have to put in here:

<img width="1022" alt="Bildschirmfoto 2024-05-25 um 13 27 55" src="https://github.com/jcwillox/hass-auto-backup/assets/32268233/197d95d2-109a-4f77-b3b6-e1401ec4150e">

The config in the example, doesn't work...

**Comment by steps56** [comment]
At my automation, the "what to do" part is:

```
metadata: {}
data:
  compressed: false
  keep_days: 2
  location: /backup
action: auto_backup.backup_full
```
The other part before is:

![Screenshot_20241107_205738_Home Assistant](https://github.com/user-attachments/assets/3071dacf-a1d7-4c34-946d-87a128102117)

Thats it 🤷🏼‍♂️.
Working always daily and nice.
Stored at the backup folder like usual.

I also use Goggle Dive backup Addon side by, to have a separate backup outside my system too.
A 2. ssd drive with a clone of the cpl. system, is always available too.
So in worst case (Hardware failure f. i.) I just plug in the 2. ssd drive instead the ssd in use (never rely at sd cards !... I use Raspi P4), reboot the system and use the latest backup from Google Drive as recovery to restore via the backup folder, to which its copied before by samba shareä.
Takes a few minutes after about 20 Minute the back copy takes by itself für 20 GB (in my case), to get a running system up to date by only max. 1 day back concerning data history and all the rest.
Saved my day a few times really 😎👌.
